chia/rpc/wallet_rpc_api.py

```diff
@@ -909,6 +910,28 @@ async def get_current_derivation_index(self, request) -> Dict[str, Any]:
 
         return {"success": True, "index": index}
 
+    async def reset_wallets_db(self, request) -> Dict[str, Any]:
```

Comment:
this name is a little bit misleading. it sounds like it's clearing the whole database.
maybe something like "reset wallet sync" would make more sense.